### ISSUE TITLE
Add abo-abo's project

### ISSUE THREAD

**Issue by azzamsa** [tarsius/elisp-maintainers#12]
Abo-abo's project was added here https://github.com/tarsius/elisp-maintainers/pull/6/files#diff-aacb4ce123f9200898dff4a35e6a8461R65. Now it's gone.

**Comment by tarsius** [comment]
No, the entry for abo-abo is still there.

**Comment by azzamsa** [comment]
![2019-01-20-03 26 04](https://user-images.githubusercontent.com/17734314/51431977-2800ec80-1c63-11e9-84b2-b50cc614b959.png)

No projects ?

**Comment by tarsius** [comment]
My mistake I thought you were talking about the whole entry.

The commit you mention was from a pr that was not merged. Could you open a new pr that adds the projects please?

**Comment by azzamsa** [comment]
ping @tarsius :)

**Comment by tarsius** [comment]
I was delaying merging this until after #13, but that has staled so I merged this now.
